### DISCUSSION TITLE
Switch Lora dynamicaly

### DISCUSSION THREAD

**Discussion by rawmen0x** [Comfy-Org/ComfyUI#1638] [Q&A]
Hi, 

Does anyone know if there is a way to switch Lora models, a big bunch, dynamicaly? Or any custom nodes you are using for that?

Trying to test on Combinatorial Prompts connecting to filename as input, but I failed as it's a drop down not a string. 

Thank you.

**Comment by ltdrdata** [comment]
You can test loras through Efficient Nodes' XY Plot.

**Reply by rawmen0x** [reply]
Thank you, also found this one, and it's working fine too: https://github.com/skfoo/ComfyUI-Coziness
